.gitignore

```diff
@@ -94,4 +94,7 @@ ENV/
 
 # Rope project settings
 .ropeproject
-.build
\ No newline at end of file
+.build
+
+# Test generated files
+git_info
```

Comment:
No new line at end of file

Reply:
didn't think that was a think for .gitignore files

Reply:
I noticed the error on here and thought it was I could be wrong